test/test_jit.py

```diff
@@ -7933,16 +7933,18 @@ def test_tensor_number_math(self):
         self._test_tensor_number_math()
 
     def test_torch_tensor_bad_input(self):
-        with self.assertRaisesRegex(RuntimeError, "Input list to torch.tensor must be of ints, floats, "
+        with self.assertRaisesRegex(RuntimeError, "must be of ints, floats, "
```

Comment:
When I find regex checks that fail because of incidental changes to the error message (e.g. one valid explanation of the error for another), I remove the part that jittered so that future changes don't have to adjust the tests.

Reply:
Seems like an odd state to leave the error message in if it's meant to be read by humans, but no idea whether this ever actually happens

Reply:
the regex is just a subset of the error message. The actual error message is readable.